analysis_template/plhld/production/cutflow_features.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+"""
+Selectors to set ak columns for cutflow features
+"""
+
+from columnflow.util import maybe_import
+from columnflow.columnar_util import set_ak_column, Route, EMPTY_FLOAT
+from columnflow.selection import Selector, SelectionResult, selector
+
+ak = maybe_import("awkward")
+
+
+@selector(
+    uses={"Jet.pt"},
+    produces={
+        "cutflow.jet1_pt", "cutflow.jet2_pt", "cutflow.jet3_pt", "cutflow.jet4_pt",
+        "cutflow.n_jet", "cutflow.n_electron", "cutflow.n_muon",
+    },
+)
+def cutflow_features(self: Selector, events: ak.Array, results: SelectionResult, **kwargs) -> ak.Array:
```

Comment:
would be nice to add the cutflow variable instances here as part of the `cutflow_features.init`

Reply:
Can you elaborate more? Not sure I understand correctly.